ui/src/app/views/resource-card/component.tsx

```diff
@@ -19,13 +19,14 @@ import {Domain, RecentResource} from 'generated/fetch';
 import {cohortsApi, conceptSetsApi, workspacesApi} from 'app/services/swagger-fetch-clients';
 import {environment} from 'environments/environment';
 
-const ResourceCardMenu: React.FunctionComponent<{
+export const ResourceCardMenu: React.FunctionComponent<{
```

Comment:
Now that this is used multiple places in different ways, we may want to move it to a common component, and maybe rename as `ResourceMenu`

Reply:
Yup and maybe to where ever I put that resourceList thing too...